tox.ini

```diff
@@ -19,29 +19,32 @@ envlist =
     docs
 
 [testenv]
-commands = pytest --lsof -ra {posargs:testing}
+commands = {env:TOX_COVERAGE_RUN:} pytest --lsof -ra {posargs:testing}
```

Comment:
can we just introduce a coverage factor for this?

Reply:
@RonnyPfannschmidt 
There is no way to have a command by default and then another for a factor, is there? (where the command comes not from the default testenv).

Reply:
Will do it via `setenv`:
```
[testenv]
commands = {env:_TOX_COVERAGE_RUN:} pytest --lsof -ra {posargs:testing}
    coverage: coverage report -m --skip-covered
setenv =
    coverage: _TOX_COVERAGE_RUN=env COVERAGE_FILE={toxinidir}/.coverage coverage run -a -m
whitelist_externals =
    coverage: env
```